centreon/www/include/configuration/configObject/host/DB-Func.php

```diff
@@ -1019,7 +1020,11 @@ function insertHostInDB($ret = array(), $onDemandMacro = null)
     updateNagiosServerRelation($hostId, $ret);
 
     $ret = $form->getSubmitValues();
-    if (isset($ret['dupSvTplAssoc']['dupSvTplAssoc']) && $ret['dupSvTplAssoc']['dupSvTplAssoc']) {
+
+    if (
+        (! empty($ret['dupSvTplAssoc']['dupSvTplAssoc'])
```

Comment:
Soirry, I made a typo in my previous suggestion ...
```suggestion
        ! empty($ret['dupSvTplAssoc']['dupSvTplAssoc'])
```